daemon/src/shared/client/cg_api.cpp

```diff
@@ -104,6 +104,20 @@ bool trap_GetEntityToken( char *buffer, int bufferSize )
 	return res;
 }
 
+void trap_SendMessage(uint8_t *buf, size_t buflen)
```

Comment:
The point of passing a vector by value here is that you could move it in the message. Whereas with the current API a copy is forced.